CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Transform nil attribute values to `log.Value` zero value instead of panicking in `go.opentelemetry.io/contrib/bridges/otellogrus`. (#6237)
 - Transform nil attribute values to `log.Value` zero value instead of panicking in `go.opentelemetry.io/contrib/bridges/otelzap`. (#6237)
 - Transform nil attribute values to `log.Value` zero value instead of `log.StringValue("<nil>")` in `go.opentelemetry.io/contrib/bridges/otelslog`. (#6246)
+- Fix request/response metrics in `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc`, as in StatsHandler `in` means response for Client and request for Server. (#6250)
```

Comment:
```suggestion
- Fix `rpc.client.request.*` metrics to measure requests instead of responses and `rpc.client.responses.*` metrics to measure responses instead of requests in `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc`. (#6250)
```

I think this would be a more user-facing changelog entry. Can anyone double-check if I have not messed anything?